nulib/meadow
# Enable Duo MFA in Meadow

# Summary 

Require MFA for Meadow

# Specific Changes in this PR
- upgrades to `ueberauth_nusso` v1.1.0


# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test

- Note: you may need to clear your browser history/data in order to see the view the Duo modal
- Visit Meadow and login
- confirm you are challenged with Duo authentication

<img width="1311" alt="Screenshot 2024-06-24 at 8 54 09 AM" src="https://github.com/nulib/meadow/assets/6372022/61352dca-10fe-4922-8791-dde5e5a303bf">


# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
